server/services/employers.ts

```diff
@@ -112,10 +112,13 @@ export const getSitesForUser = async (user: HcapUserInfo): Promise<EmployerSite[
     additionalCriteriaParams.userSites = user.sites;
   }
 
-  if (user.isHA && user.regions.length > 0) {
+  if (user.regions.length > 0) {
```

Comment:
An MoH will not enter this condition as they have no regions. Is that intentional?

Reply:
Not entering this condition means querying for all regions for MoH.